### ISSUE TITLE
Can't assign folder path with special characters

### ISSUE THREAD

**Issue by Avastgard** [dumbeau/AutoHotPie#27]
I assigned a slice to open a certain folder, but when I try to open it, I get the following error:

```
---------------------------
PieMenu.ahk
---------------------------
Cannot find directory:
C:\Users\user\OneDrive - NÃ‰RY\Ãrea de Trabalho\FlÃ¡vio\AutoHotkey
---------------------------
OK   
---------------------------

```

I presume this is because the file path has special characters. Is that right? Is there a way for me to manually correct this?

**Comment by dumbeau** [comment]
Could you paste what the intended path is including special characters here?  I may be able to account for this on my end.

You can export the settings file (File > Export Settings) edit that json file and then import it.

**Comment by Avastgard** [comment]
This is what it is supposed to look like:

`C:\Users\flima\OneDrive - GERALDO NÉRY LOPES ADVOGADOS ASSOCIADOS\Área de Trabalho\Flávio\AutoHotkey`

Checking the .json file, this is what was there:

`C:\\Users\\flima\\OneDrive - GERALDO NÉRY LOPES ADVOGADOS ASSOCIADOS\\Área de Trabalho\\Flávio\\AutoHotkey`

Changing the double backslashes to single ones had no effect and produced the same error message.

**Comment by dumbeau** [comment]
Thanks!  Looking into this now, seems like AHK just doesn't handle special characters in filepaths on its own, but everything on the frontend is working fine.  This should be fixable on my end.

**Comment by Avastgard** [comment]
Great, thanks for the feedback!

**Comment by dumbeau** [comment]
Still working on this, for the time being all I'd say is don't use special characters in filepaths, but I plan on having this fixed before any "official" release.

In case anyone can assist, the issue is the [JSON.ahk](https://github.com/cocobelgica/AutoHotkey-JSON) library I'm using to read json files.  Special characters in value strings get misread.  Currently I'm trying to modify this library to fix this, unless there is another AHK library out there to handle this.

**Comment by Avastgard** [comment]
That's great to hear! I can change file paths in my own PC, but since this script will be particularly helpful to me at work, and I can't change file paths there, knowing this will eventually be addressed is nice. Thanks for this great script!

**Comment by dumbeau** [comment]
omg the solution ended up being really simple.  This is fixed in [v1.0.17-alpha](https://github.com/dumbeau/AutoHotPie/releases).

**Comment by Avastgard** [comment]
Just tested and it works! Thank you very much!
